Xamarin.Forms.Core.UnitTests/ViewUnitTests.cs

```diff
@@ -282,7 +282,7 @@ public void TestFadeTo ()
 			var view = new View {IsPlatformEnabled = true, Platform = new UnitPlatform ()};
 			Ticker.Default = new BlockingTicker ();
 
-			view.FadeTo (0.1);
+            view.FadeTo (0.1, finished:() => Console.WriteLine("TestFadeTo.Finished"));
```

Comment:
This _demonstrates_ the callback, but it doesn't _test_ that the callback is invoked (i.e., this test wouldn't fail even if `finished?.Invoke();` were missing from the `FadeTo` method). This needs a test which would fail if the callback weren't invoked. And all of the other updated methods would need similar tests.